web/src/beta/components/TabMenu/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { FC, useState, ReactNode } from "react";
+
+import Icon from "@reearth/beta/components/Icon";
+import Icons from "@reearth/beta/components/Icon/icons";
+import { styled } from "@reearth/services/theme";
+
+interface TabObject {
+  icon: keyof typeof Icons;
+  component: ReactNode;
+}
+
+type Props = {
+  title: string;
+  tabs: Record<string, TabObject>;
```

Comment:
If you make tabs just an array of objects, you can map from tabs itself instead of doing `Object.entries.....`

Reply:
I wanted to use objects rather than array so it's easier to select the tabs with a <key, value> config. If you think arrays are better, I'll switch to that!

Reply:
When selecting a tab it's also a better than using `index` or `find` to set the right tab especially if we are handling the `setting` outside this component

Reply:
Thats fair. If this wasn't a prop field I'd say keep it as a record, but since its a prop, I feel like we should keep a similar syntax standard and up until now (and seen in many UI libraries, like Ant Design) we have done an array of objects that we iterate over.

Reply:
Fair. Will make the change to Array!

Reply:
This is done!